.circleci/config.yml

```diff
@@ -2009,7 +2036,7 @@ jobs:
           checkout-and-assume-cache: true
           attach: false
 
-  mas-testing-gn-check:
+  mas-testing-x64-gn-check:
```

Comment:
do we need to build MAS versions of arm64 as well?

Reply:
Yes but not right now